packages/template-retail-react-app/app/ssr.js

```diff
@@ -21,7 +21,7 @@ const options = {
 
     // The path to the favicon. This must also appear in
     // the mobify.ssrShared section of package.json.
-    faviconPath: path.resolve(process.cwd(), 'build/static/ico/favicon.ico'),
+    faviconPath: path.resolve(process.cwd(), 'app/static/ico/favicon.ico'),
```

Comment:
Will this work on prod?

Reply:
good call. @shethj do you wanna deploy this to a managed runtime environment and check?

Reply:
I added a conditional to set favicon url based on whether the app is being served in development or from a build.